eng/common/scripts/Verify-Links.ps1

```diff
@@ -65,11 +67,11 @@ function ResolveUri ([System.Uri]$referralUri, [string]$link)
   }
 
   $linkUri = [System.Uri]$link;
-  if($resolveRelativeLinks){
+  if($linkGuidance){
```

Comment:
```suggestion
  if($linkGuidance) {
```

nit: space before brace

Reply:
Will make the change together with renaming.